test/topology_custom/test_boot_time.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+import pytest
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_boot(manager):
+    rbno = True
+    cfg = {'enable_repair_based_node_ops': rbno}
+    await manager.server_add(config=cfg)
+
+    for i in range(3):
+        logger.info(f"booting node {i+1}")
+        start = time.time()
+        await manager.server_add(config=cfg)
+        end = time.time()
+        logger.info(f"node {i+1} took {end - start}ms to boot")
```

Comment:
Is this test useful as a unit-test? It doesn't seem to test anything, there is no fail criteria.
I understand this was useful to develop this series, but I don't think it is worth including among the automatically run tests, no one will look at the boot times.

Reply:
It tests the bootstrap does not fail, in addition it also gets the bootstrap time.

Reply:
I will drop the test for now.